runtime/drivers/bigquery/bigquery.go

```diff
@@ -95,7 +95,7 @@ type configProperties struct {
 	AllowHostAccess bool   `mapstructure:"allow_host_access"`
 }
 
-func (d driver) Open(config map[string]any, logger *zap.Logger) (drivers.Connection, error) {
```

Comment:
I think it should add a check and return an error if `shared` is `true` (since it doesn't support that). All the drivers that can't be shared should do that (only `sqlite` can be shared right now, I think?)

Reply:
Postgres can't be shared either but its not being used in runtime now I guess.

Reply:
Yeah we should probably just remove Postgres